app/utils/simplified-account/validation/responsible-person.schema.js

```diff
@@ -0,0 +1,146 @@
+const { body } = require('express-validator')
+const { formatPhoneNumberWithCountryCode } = require('../../telephone-number-utils')
+
+const responsiblePersonSchema = {
+  name: {
+    firstName: {
+      validate: body('firstName')
+        .notEmpty()
+        .withMessage('Enter your first name')
+        .bail()
+        .isLength({ max: 100 })
+        .withMessage('First name must be 100 characters or fewer')
+    },
+    lastName: {
+      validate: body('lastName')
+        .notEmpty()
+        .withMessage('Enter your last name')
+        .bail()
+        .isLength({ max: 100 })
+        .withMessage('Last name must be 100 characters or fewer')
+    }
+  },
+  dob: {
+    validate: body('dobDay') // Use just one DOB field as the anchor to prevent multiple validations
+      .custom((value, { req }) => {
+        const { dobDay, dobMonth, dobYear } = req.body
+        if (!dobDay && !dobMonth && !dobYear) {
+          throw new Error('Enter the date of birth')
+        }
+
+        // Stripe requires minimum age of 13
+        const day = Number(dobDay)
```

Comment:
Does the [dobDay validation](https://github.com/alphagov/pay-selfservice/pull/4323/files#diff-2bd10cbbf12a432d034d00530df64d47e0663f73ad0032e46eb5226ae4ad145dR52) get executed before this line gets executed?

Reply:
no, they are run in sequence with some conditions on the individual inputs to only run if some values are already set